ResearchKit/Consent/ORKConsentReviewStepViewController.m

```diff
@@ -377,6 +377,7 @@ - (void)goToPage:(NSInteger)page animated:(BOOL)animated {
         if (finished) {
             ORKStrongTypeOf(weakSelf) strongSelf = weakSelf;
             [strongSelf updateBackButton];
+            [[strongSelf consentReviewDelegate] consentReviewStepViewController:strongSelf didShowPhaseIndex:page];
```

Comment:
Minor - since this is Objective C, perhaps a check to `respondsToSelector:(SEL)` before calling, just in case?

Reply:
Agreed. I added respondsToSelector calls for both cases.